Cargo.toml

```
@@ -20,3 +20,4 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 tar = "0.4.30"
 pico-args = "0.3.4"
+indicatif = "0.16.0"
```

```suggestion
indicatif = { version = "0.16.0", default-features = false }
```
This drops some unicode stuff we don't need because everything we display is ASCII, and removes a couple of people from the supply chain.

---

Hang on, `default-features = false` does nothing even though it should do something useful. I'll look into this further at a later date, don't want to hold up this PR.

---

Ah I've figured it out, it's indicatif's fault: https://github.com/mitsuhiko/indicatif/issues/280